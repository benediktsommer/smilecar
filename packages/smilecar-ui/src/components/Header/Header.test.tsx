import { render } from '@testing-library/react';

import { Header } from './';

describe('Test <Header />', () => {
  it('should render an html markup', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
