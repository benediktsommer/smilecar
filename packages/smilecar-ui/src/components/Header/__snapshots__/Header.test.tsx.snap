// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Header /> should render an html markup 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      class="MuiBox-root MuiBox-root-1"
    >
      <div
        class="MuiBox-root MuiBox-root-2"
      >
        <a
          href="/"
        >
          <svg
            class="logo1024"
            height="44"
            viewBox="0 0 278 44"
            width="278"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
            >
              <g>
                <path
                  d="M167.934 4.828h-4.835v6.206h4.71c.514.01 1.025-.067 1.513-.227 1.23-.43 2.014-1.635 1.906-2.931 0-2.959-2.293-3.02-3.287-3.02m3.287 18.51l-4.558-8.125h-3.544v8.118h-5.74V.6h10.25c.65 0 1.292 0 1.935.034 1.036.063 4.648.29 6.416 3.538.606 1.175.917 2.48.905 3.8.103 1.727-.54 3.414-1.768 4.635-.322.327-.679.618-1.064.869-.593.308-1.207.575-1.837.8l5.256 9.083-6.25.007zM143.544 6.069l-2.804 9.593h5.768l-2.964-9.593zm5.388 17.317l-1.119-3.641h-8.392l-1.133 3.641h-6.057L140.678.628h5.926l8.386 22.758h-6.058zM129.937 20.393c-2.12 2.366-5.143 3.726-8.323 3.745-1.596-.005-3.176-.333-4.641-.966-5.996-2.703-6.286-9.303-6.286-11.11-.05-2.942.926-5.81 2.763-8.11 1.982-2.537 5.04-4 8.261-3.952 3.062-.027 5.988 1.262 8.033 3.538.606.727 1.134 1.515 1.575 2.352l-4.483 2.22c-.208-.426-.456-.833-.739-1.213-.943-1.36-2.488-2.178-4.144-2.194-1.286.003-2.522.496-3.454 1.38-.932.862-2.162 2.758-2.162 5.89-.005.516.03 1.033.104 1.544.387 3.117 1.733 4.538 2.997 5.249.765.435 1.634.657 2.515.64 3.19 0 4.413-2.475 4.931-3.35l4.518 2.289c-.46.714-.955 1.405-1.485 2.069"
                  fill="#2479D3"
                  transform="translate(58.02 9.655)"
                />
                <path
                  d="M103.939 12.393L103.939 23.255 98.275 23.255 98.275 12.559 89.648.31 96.224.31 101.169 8.076 106.087.31 112.628.31zM61.702 5.8c-.912-1.269-2.57-1.269-3.937-1.269H54.09v6.952h3.453c.654.03 1.308-.003 1.955-.097 2.017-.317 2.825-1.558 2.763-3.503.035-.74-.185-1.47-.622-2.07m4.946 6.663c-2.148 3.055-5.401 3.283-8.717 3.283h-3.84v7.49H48.35V.31h11.39c2.183 0 4.234.262 5.988 1.787.4.325.75.709 1.037 1.137.855 1.409 1.295 3.03 1.27 4.676.008 1.622-.473 3.208-1.381 4.552M83.715 5.8c-.905-1.269-2.57-1.269-3.937-1.269h-3.675v6.952h3.454c.653.03 1.307-.003 1.954-.097 2.01-.317 2.832-1.558 2.763-3.503.032-.74-.187-1.469-.621-2.07m4.945 6.663c-2.148 3.055-5.401 3.283-8.717 3.283h-3.84v7.49h-5.76V.31h11.39c2.182 0 4.226.262 5.988 1.787.4.325.749.709 1.036 1.137.855 1.409 1.295 3.03 1.27 4.676.008 1.622-.473 3.208-1.38 4.552M34.287 5.772l-2.832 9.656H37.3l-3.012-9.656zm5.436 17.476l-1.14-3.669h-8.468l-1.105 3.67h-6.147L31.386.302h5.989l8.468 22.945h-6.12zM14.187 23.255L14.187 13.469 5.795 13.469 5.795 23.255 0 23.255 0 .303 5.823.303 5.823 8.593 14.187 8.593 14.187.303 19.955.303 19.955 23.255z"
                  fill="#E85922"
                  transform="translate(58.02 9.655)"
                />
              </g>
              <path
                d="M43.515 38.214V29.51c-6.385 4.462-13.992 6.85-21.785 6.842-7.77.007-15.356-2.369-21.73-6.807v8.669c.004 2.892 2.353 5.234 5.25 5.234h33.016c2.896 0 5.245-2.342 5.25-5.234z"
                fill="#2479D3"
              />
              <path
                d="M43.515 38.214V29.51c-6.385 4.462-13.992 6.85-21.785 6.842-7.77.007-15.356-2.369-21.73-6.807v8.669c.004 2.892 2.353 5.234 5.25 5.234h33.016c2.896 0 5.245-2.342 5.25-5.234z"
                fill="#E85922"
              />
              <path
                d="M38.266 0H5.249C3.857-.002 2.52.55 1.535 1.533.551 2.516-.002 3.851 0 5.241v24.304c6.374 4.438 13.96 6.814 21.73 6.807 7.793.009 15.4-2.38 21.785-6.842V5.241c.002-1.39-.55-2.725-1.535-3.708C40.995.55 39.658-.002 38.266 0z"
                fill="#2479D3"
              />
              <path
                d="M38.266 0H5.249C3.857-.002 2.52.55 1.535 1.533.551 2.516-.002 3.851 0 5.241v24.304c6.374 4.438 13.96 6.814 21.73 6.807 7.793.009 15.4-2.38 21.785-6.842V5.241c.002-1.39-.55-2.725-1.535-3.708C40.995.55 39.658-.002 38.266 0z"
                fill="#2479D3"
              />
              <path
                d="M33.797 13.234s-1.016-5.262-1.195-6.206c-.264-1.272-1.38-2.188-2.68-2.2H13.593c-1.322.018-2.445.97-2.68 2.269l-1.195 6.206c-2.03.738-3.91 2.76-3.91 5.138v6.78c0 1.255 1.237 2.31 2.35 2.613v2.945c0 .678.55 1.228 1.229 1.228h2.977c.679 0 1.23-.55 1.23-1.228v-2.91h16.32v2.834c0 .682.554 1.235 1.237 1.235h2.97c.68-.004 1.23-.555 1.23-1.235v-2.93c1.112-.31 2.355-1.38 2.355-2.621v-6.78c0-2.372-1.879-4.4-3.91-5.138zm-22.573 7.76c-1.453 0-2.632-1.177-2.632-2.628 0-1.452 1.179-2.628 2.632-2.628 1.454 0 2.632 1.176 2.632 2.628-.004 1.45-1.18 2.623-2.632 2.627zM29.64 23.71c-.056.424-.415.742-.843.745h-.097c-.224-.026-.43-.14-.57-.317-.14-.178-.203-.403-.176-.628v-.241l-.117.097c-2.022 1.018-4.238 1.594-6.5 1.69-2.554 0-5.048-.777-7.149-2.228-.358-.266-.462-.757-.241-1.145.087-.184.25-.32.448-.372.197-.052.407-.014.574.103 1.877 1.29 4.103 1.978 6.382 1.972 2.043-.091 4.044-.613 5.871-1.53.043-.016.087-.027.131-.035-.098-.112-.21-.211-.331-.297-.276-.156-.442-.452-.43-.769.012-.316.199-.6.485-.735.286-.136.625-.1.877.09.598.384 1.076.926 1.382 1.566.283.638.388 1.341.304 2.034zm2.652-2.717c-1.453 0-2.632-1.176-2.632-2.627 0-1.452 1.179-2.628 2.632-2.628s2.632 1.176 2.632 2.628c0 1.45-1.179 2.627-2.632 2.627z"
                fill="#FFF"
              />
            </g>
          </svg>
        </a>
      </div>
      <div
        class="MuiBox-root MuiBox-root-3"
      >
        <div>
          <input
            checked=""
            name="language"
            type="radio"
            value="de"
          />
           Deutsch
          <input
            name="language"
            type="radio"
            value="en"
          />
           English
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
